modules/upi-limitations-for-scaling-a-upi-cluster.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/scaling-a-user-provisioned-cluster-with-the-bare-metal-operator.adoc
+:_content-type: CONCEPT
+
+[id="upi-limitations-for-scaling-a-upi-cluster_{context}"]
+= Limitations for scaling a user-provisioned cluster
+
+* You cannot use a provisioning network to scale user-provisioned infrastructure clusters by using the Bare Metal Operator (BMO).
```

Comment:
As a consequence of this limitation, we only support BareMetalHost drivers that use virtual media, for example `redfish-virtualmedia` (but not `redfish` or `ipmi`). Worth mentioning explicitly?

Reply:
yes I will take a look at this. Please let me know if any other limitations or requirements you can think of too.

Reply:
I think this is the biggest.